components/CrsSelector.vue

```diff
@@ -24,6 +24,7 @@
 </template>
 
 <script setup>
+  import schema from "@geode/opengeodeweb-front/assets/schemas/CrsSelector.json"
```

Comment:
You're already in OGW-Front, just put "@/assets/schemas/schema.json" in every component